sdk/tools/gen_baseaddress.py

```diff
@@ -220,10 +223,14 @@
     IMAGE_NT_OPTIONAL_HDR64_MAGIC: 0
 }
 
+IS_64_BIT = False
+
 def is_x64():
```

Comment:
I don't see anymore the value of keeping this function. Couldn't we just use the `IS_64_BIT` variable wherever that function was previously used?

Reply:
Removed it.